version.py

```diff
@@ -34,7 +34,8 @@
 # new level means in api_docs/changelog.md, as well as "**Changes**"
 # entries in the endpoint's documentation in `zulip.yaml`.
 
-API_FEATURE_LEVEL = 292  # Last bumped for `namedusergroup_creator_date_created`.
+
+API_FEATURE_LEVEL = 293  # Last bumped for `allow_private_data_export` setting.
```

Comment:
Don't add extra blank lines here.

Reply:
Ah, I think this got added while resolving merge conflicts.